proto/cosmos/gov/v1beta1/gov.proto

```diff
@@ -151,22 +151,24 @@ message DepositParams {
   //  Minimum deposit for a proposal to enter voting period.
   repeated cosmos.base.v1beta1.Coin min_deposit = 1 [
     (gogoproto.nullable)     = false,
-    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (gogoproto.jsontag)      = "min_deposit,omitempty"
   ];
 
   //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2
   //  months.
   google.protobuf.Duration max_deposit_period = 2 [
     (gogoproto.nullable)    = false,
-    (gogoproto.stdduration) = true
+    (gogoproto.stdduration) = true,
+    (gogoproto.jsontag)     = "max_deposit_period,omitempty"
   ];
 }
 
 // VotingParams defines the params for voting on governance proposals.
 message VotingParams {
   //  Length of the voting period.
   google.protobuf.Duration voting_period = 1
-      [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+      [(gogoproto.nullable) = false, (gogoproto.stdduration) = true, json_name = "voting_period"];
```

Comment:
This had `(gogoproto.jsontag)     = "voting_period,omitempty",` before, and I think omitempty might be important